rules_building_block/discovery_net_share_discovery_winlog.toml

```diff
@@ -28,10 +28,10 @@ building_block_type = "default"
 query = '''
 sequence by user.name, source.port, source.ip with maxspan=15s 
  [file where event.action == "network-share-object-access-checked" and 
-  winlog.event_data.ShareName : ("\\*ADMIN$", "\\*C$") and 
+  winlog.event_data.ShareName in ("\\\\*\\ADMIN$", "\\\\*\\C$") and 
   source.ip != null and source.ip != "0.0.0.0" and source.ip != "::1" and source.ip != "::" and source.ip != "127.0.0.1"]
  [file where event.action == "network-share-object-access-checked" and 
-  winlog.event_data.ShareName : ("\\*ADMIN$", "\\*C$") and 
+  winlog.event_data.ShareName in ("\\\\*\\ADMIN$", "\\\\*\\C$") and 
```

Comment:
```suggestion
  winlog.event_data.ShareName : ("\\\\*\\ADMIN$", "\\\\*\\C$") and 
```

Reply:
Wildcards do not work with `in`, so I don't think this query will capture anything anymore.

Reply:
That's the intent here, we need to match the `\\*\ADMIN$` & `\\*\C$` values, literal wildcards